blocks/library/subhead/index.js

```diff
@@ -60,28 +60,19 @@ export const settings = {
 	edit( { attributes, setAttributes, isSelected, className } ) {
 		const { content, placeholder } = attributes;
 
-		return [
-			isSelected && (
-				<InspectorControls key="inspector">
-					<BlockDescription>
-						<p>{ __( 'Explanatory text under the main heading of an article.' ) }</p>
-					</BlockDescription>
-				</InspectorControls>
-			),
-			<RichText
-				tagName="p"
-				key="editable"
-				value={ content }
-				onChange={ ( nextContent ) => {
-					setAttributes( {
-						content: nextContent,
-					} );
-				} }
-				className={ className }
-				placeholder={ placeholder || __( 'Write subhead…' ) }
-				isSelected={ isSelected }
-			/>,
-		];
+		return <RichText
```

Comment:
In general for multi-line returned elements I prefer something like this:

```js
return (
  <Component
    prop="prop"
    prop2="prop2"
  />
);
```

Reply:
@youknowriad Thanks, you're right, thats more readable. I added the parentheses.